src/BloomExe/WebLibraryIntegration/BloomS3Client.cs

```diff
@@ -158,7 +159,11 @@ public void UploadBook(string storageKeyOfBookFolder, string pathToBloomBookDire
 
 			Directory.CreateDirectory(wrapperPath);
 
-			CopyDirectory(pathToBloomBookDirectory, Path.Combine(wrapperPath, Path.GetFileName(pathToBloomBookDirectory)));
+			var destDirName = Path.Combine(wrapperPath, Path.GetFileName(pathToBloomBookDirectory));
+			CopyDirectory(pathToBloomBookDirectory, destDirName);
+			var unwantedPdfs = Directory.EnumerateFiles(destDirName, "*.pdf").Where(x => Path.GetFileName(x) != pdfToInclude);
```

Comment:
The unit test seems to indicate that something else is already preventing upload of thumbs.db, though I don't know what.